src/containers/blocks.jsx

```diff
@@ -125,6 +125,8 @@ class Blocks extends React.Component {
         if (this.props.isVisible) { // Scripts tab
             this.workspace.setVisible(true);
             this.props.vm.refreshWorkspace();
+            // Re-enable toolbox refreshes without causing one. See #updateToolbox for more info.
+            this.workspace.toolboxRefreshEnabled_ = true;
```

Comment:
post merge +1

I had assumed that anytime we needed to `refreshWorkspace` it would also be causing us to change the `toolboxXML` prop which would cause an `updateToolbox` -- thanks for catching this bad assumption here!